docs/tutorials/optimizers_conditionalgradient.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>Can you make the Run in Colab link to <code>https://colab.research.google.com/github/tensorflow/addons/blob/master/docs/tutorials/optimizers_conditionalgradient.ipynb</code></p><p> </p><p>The link will work after it merges.</p><p> </p><p>The Tensorflow org link can be:</p><p><code>https://www.tensorflow.org/addons/tutorials/optimizers_conditionalgradient</code></p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>